rollup.config.js

```diff
@@ -0,0 +1,103 @@
+import babel from 'rollup-plugin-babel';
+import html from 'rollup-plugin-html';
+import json from 'rollup-plugin-json';
+import pkg from './package.json';
+import uglify from 'rollup-plugin-uglify';
+import { minify } from 'uglify-es';
+import less from './lib/rollup-plugin-less';
+
+let htmlPlugin = html ({
+    include: [
+        '**/*.svg',
+        '**/*.html',
+    ],
+    htmlMinifierOptions: {
+        collapseWhitespace: true,
+        quoteCharacter: "'", // reduces escape characters
+    },
+});
+
+let babelPlugin = babel({
+    exclude: 'node_modules/**/*',
```

Comment:
Is this using `.babelrc`?

Reply:
It is, but the `exclude` option is for the plugin itself, not for babel. Babel doesn't support the `exclude` option.